src/explorer/ledgerexplorer.ts

```diff
@@ -24,24 +24,6 @@ const requestInterceptor = (request: AxiosRequestConfig): AxiosRequestConfig =>
   return request;
 };
 
-const responseInterceptor = (
```

Comment:
Why removing the definition here? You can make your changes here, define an additional `const errorInterceptor`, then you can call `client.interceptors.response.use(responseInterceptor, errorInterceptor);`

Reply:
Fixed